main.js

```diff
@@ -0,0 +1,45 @@
+const toDoList = [];
+const taskInput = document.querySelector(".add input");
+const taskCounter = document.querySelector("h1 span");
+const ul = document.querySelector("ul");
+
+const renderList = () => {
+  ul.textContent = "";
```

Comment:
Ja bym wydzielił to do oddzielnej funkcji, jak nazwiesz ją jakoś deskryptywnie `removeAllTasks` czy `clearList` to będzie się dużo lepiej kod czytać